packages/extension-koni/public/locales/en/translation.json

```diff
@@ -1,5 +1,163 @@
+  "settings.security.Screen.changePassword.confirmAgreement": "I understand that SubWallet can’t recover the password.",
+  "settings.security.Screen.changePassword.link.learnMore": "Learn more",
+  "settings.Screen.securitySetting.Selector.autoLock.title": "Auto lock",
+  "settings.Screen.securitySetting.Selector.autoLock.itemLabel": "{{time}} minutes",
+  "settings.Screen.securitySetting.Selector.authPassword.Item.alwaysRequired": "Always required",
+  "settings.Screen.securitySetting.Selector.authPassword.Item.whenNeeded": "When needed",
+  "settings.Screen.manageWebsiteAccess.Filter.substrate": "Substrate dApp",
+  "settings.Screen.manageWebsiteAccess.Filter.ethereum": "Ethereum dApp",
+  "settings.Screen.manageWebsiteAccess.Filter.blocked": "Blocked dApp",
+  "settings.Screen.manageWebsiteAccess.Filter.connected": "Connected dApp",
+  "settings.manageWebsiteAccess.Modal.configuration.Item.forget": "Forget all",
+  "settings.manageWebsiteAccess.Modal.configuration.Item.disconnect": "Disconnect all",
+  "settings.manageWebsiteAccess.Modal.configuration.Item.connect": "Connect all",
+  "settings.manageWebsiteAccess.Modal.accessConfiguration.title": "Access configuration",
+  "settings.Screen.manageNetworks.Filter.evm": "EVM networks",
+  "settings.Screen.manageNetworks.Filter.substrate": "Substrate networks",
+  "settings.Screen.manageNetworks.Filter.custom": "Custom networks",
+  "settings.Screen.manageNetworks.Filter.enabled": "Enabled networks",
+  "settings.Screen.manageNetworks.Filter.disabled": "Disabled networks",
+  "settings.Screen.manageNetworks.importNetworkModal.title": "Import network",
+  "settings.Screen.manageNetworks.importNetworkModal.bodyTitle": "Currently support WSS provider for Substrate networks and HTTP provider for EVM network",
+  "settings.Screen.manageTokens.Filter.enabled": "Enabled tokens",
+  "settings.Screen.manageTokens.Filter.disabled": "Disabled tokens",
+  "settings.Screen.manageTokens.Filter.custom": "Custom tokens",
+  "settings.Screen.manageTokens.importTokens.title": "Import token",
+  "settings.Screen.manageTokens.importTokens.button.submit": "Import token",
+  "settings.Screen.manageTokens.importTokens.contractPlaceHolder": "Please type or paste an address",
+  "settings.manageAddressBook.Modal.addContact.title": "Add contact",
+  "settings.manageAddressBook.Modal.addContact.Input.name.label": "Contact name",
+  "settings.manageAddressBook.Modal.addContact.Input.address.label": "Contact address",
+  "settings.manageAddressBook.Filter.saveContact": "Saved contacts",
+  "settings.manageAddressBook.Filter.recent": "Recent",
+  "settings.manageAddressBook.Filter.yourWallet": "Your wallet",
+  "settings.aboutSubwallet.Modal.title": "About SubWallet",
+  "settings.aboutSubwallet.Modal.ToS": "Terms of use",
+  "settings.aboutSubwallet.Modal.website": "Website",
+  "settings.aboutSubwallet.Modal.xTwitter": "X (Twitter)",
```

Comment:
@dungnguyen-art : check this key.
You may change it to settings.Modal.aboutSubwallet.xTwitter